CHANGELOG.md

```diff
@@ -17,10 +17,14 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
-## [Unreleased]
+## [7.73.0] - 2025-01-19
 ### Added
 - Support for the `/simulators` and `/simulators/integration` API endpoints.
```

Comment:
Lets keep the simulator stuff at the top as part of the "unreleased" (although parts are technically released)